packages/manager/modules/pci/src/projects/project/public-gateways/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,39 @@
+  "pci_projects_project_public_gateways_add_spec_publiccloud-gateway-m_size": "Taille M",
+  "pci_projects_project_public_gateways_add_spec_publiccloud-gateway-m_bandwidth": "Jusqu'à {{bandwidth}}",
+  "pci_projects_project_public_gateways_add_spec_publiccloud-gateway-m_price": "<strong>Prix:</strong> {{price}}",
+  "pci_projects_project_public_gateways_add_spec_publiccloud-gateway-l_size": "Taille L",
+  "pci_projects_project_public_gateways_add_spec_publiccloud-gateway-l_bandwidth": "Jusqu'à {{bandwidth}}",
+  "pci_projects_project_public_gateways_add_spec_publiccloud-gateway-l_price": "<strong>Prix:</strong> {{price}}",
+  "pci_projects_project_public_gateways_add_region_title": "Sélectionner une localisation",
+  "pci_projects_project_public_gateways_add_region_info": "Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des fonctions par région, <a href=\"{{url}}\" target='_blank'>cliquez ici.</a>",
+  "pci_projects_project_public_gateways_add_attach_private_network_to_public_gateway_sub_title": "Attacher à un réseau privé spécifique",
+  "pci_projects_project_public_gateways_add_attach_private_network_to_public_gateway_intro1": "Pour utiliser Public Gateway, vous devez sélectionner un réseau privé auquel vous connecter. Les instances de ce dernier pourront accéder à Internet de manière sécurisée à l'aide de cette Public Gateway.",
+  "pci_projects_project_public_gateways_add_attach_private_network_to_public_gateway_intro2": "En option, un nouveau réseau privé peut être créé pour être utilisé avec celle-ci.",
+  "pci_projects_project_public_gateways_add_select_private_network": "Sélectionner un réseau privé",
+  "pci_projects_project_public_gateways_add_private_network_add_action": "Ajouter un réseau privé",
+  "pci_projects_project_public_gateways_add_submit_label": "Créer une Public Gateway",
+  "pci_projects_project_public_gateways_add_no_private_network_warning": "Il n'existe aucun réseau privé dans la région sélectionnée. Veuillez en créer un en utilisant l'un des 3 subnets suivants.",
+  "pci_projects_project_public_gateways_add_modal_title": "Créer un réseau privé",
+  "pci_projects_project_public_gateways_add_modal_description": "Créez un réseau privé et avec un subnet dans lequel vous souhaitez utiliser votre Public Gateway.",
+  "pci_projects_project_public_gateways_add_modal_add_private_network_field_label": "Nom du réseau privé",
+  "pci_projects_project_public_gateways_add_modal_add_private_network_field_placeholder": "Saisir un nom de réseau privé",
+  "pci_projects_project_public_gateways_add_modal_add_private_network_select_label": "Sélectionnez un subnet",
+  "pci_projects_project_public_gateways_add_modal_submit_label": "Ajouter",
+  "pci_projects_project_public_gateways_add_modal_cancel_label": "Annuler",
+  "pci_projects_project_public_gateways_add_modal_add_private_network_error": "{{message}}",
```

Comment:
**question:** Why do we need to translation to display the message returned by server.

Reply:
@anooparveti I forgot to add the translated value here for this key, after value should show the error message.